notificationdemo/source/android/app/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "org.kaaproject.demo.notification"
+        minSdkVersion 15
+        targetSdkVersion 23
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+
+    sourceSets {
+        main {
+            manifest.srcFile 'src/main/AndroidManifest.xml'
+        }
+    }
+}
+
+dependencies {
+    compile files('libs/kaa-android-ep-sdk-FGMjC8ayctXgea_Fy75BVWuphaI.jar')
```

Comment:
Are you sure that this works (build directly from the sandbox)? Looks like hardcoded value.